biz.aQute.bndlib/src/aQute/bnd/osgi/Clazz.java

```diff
@@ -479,6 +480,21 @@ public TypeRef getType() {
 			return type;
 		}
 
+		public int getModuleAccess() {
+			return attribute(ModuleAttribute.class).map(attr -> attr.module_flags)
```

Comment:
Use `mapToInt` instead of `map`. To do that, you will need to use `attributes`.

```java
			return attributes(ModuleAttribute.class).mapToInt(attr -> attr.module_flags)
				.findFirst()
				.orElse(0);
```

Reply:
Also, this method and the one on Clazz are better named `getModuleFlags`. Since this not the access https://docs.oracle.com/javase/specs/jvms/se11/html/jvms-4.html#jvms-4.7.25

Reply:
kk, will do